pkg/apis/apps/v1alpha1/apimanager_types.go

```diff
@@ -342,6 +342,13 @@ type SystemRedisPersistentVolumeClaimSpec struct {
 type SystemPVCSpec struct {
 	// +optional
 	StorageClassName *string `json:"storageClassName,omitempty"`
+	// Resources represents the minimum resources the volume should have.
+	// Ignored when VolumeName field is set
```

Comment:
About this...

Unfortunately, although Resources is ignored when VolumeName field is set, it seems that it is still mandatory at the PVC level even if you define VolumeName in the PVC
Fortunately I think in this implementation when StorageClassName is not set we still set defaults at PVC level right? If that's the case then this would be correct I think

Reply:
Indeed, resources are always set.

Reply:
:+1: